sdk/src/main/java/ly/count/android/sdk/messaging/CountlyPush.java

```diff
@@ -434,6 +429,15 @@ public void call(Bitmap bitmap) {
 
     }
 
+    private static Intent getPushActivityIntent(final Context context, final Message msg, final Intent notificationIntent, int index) {
```

Comment:
replace name with "createPushActivityIntent".
Pass the lists directly to the function.